src/main/java/com/liferay/ide/idea/ui/modules/ext/LiferayModuleExtBuilder.java

```diff
@@ -136,9 +136,6 @@ private void _createProject(VirtualFile projectRoot, Project project) {
 		sb.append("-m ");
 		sb.append(_originalModuleName);
 
-		sb.append(" -M ");
-		sb.append(_originalModuleVersion);
```

Comment:
This need only contains part of implementation we did for ext project wizard in IDE.  Ext wizard can't show target platform depenedency artifact with new gradle workspace plugin(we added liferay.worskpace.product support). This ticket also include other implementation we did in IDE 3.9M1. Please check and resend them.